config/config.exs

```diff
@@ -27,3 +27,18 @@ import_config "#{Mix.env}.exs"
 config :phoenix, :generators,
   migration: true,
   binary_id: false
+
+# Configure plug for json mime type
+# will require plug to be recompiled
+config :plug, :mimes, %{
+  "application/vnd.api+json" => ["json-api"]
+}
+
+config :guardian, Guardian,
+  allowed_algos: ["HS512"], # optional
+  verify_module: Guardian.JWT,  # optional
+  issuer: "Yggdrasil",
+  ttl: { 1, :days },
+  verify_issuer: true, # optional
+  secret_key: "5t7ODk6J7ixv/64cq5hAObI8PRnzp334PYcGgt15ktmI2F4XcpaAHGDvwvT3sgEp",
```

Comment:
This is normal notice Phoenix config above does this too. 

https://github.com/voluspa/yggdrasil/blob/json.api/config/prod.exs

Notice the import of prod secret file at the bottom, in there you would override this with the prod secret and the git ignore has been setup by Phoenix generator to ignore that file. 

Thought the same thing when I first saw this. But they have already thought it out.